FFmpegInteropx.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <description>FFmpeg decoding library for Windows 10 UWP Apps</description>
+    <authors>FFmpegInteropX</authors>
+    <projectUrl>https://github.com/ffmpeginteropx/FFmpegInteropX</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+  <files>
+
+    <file src="ffmpeg\Build\Windows10\x64\include\**\*.h" target="include" />
+    <file src="x64\Release\FFmpegInterop\FFmpegInterop.winmd" target="lib\uap10.0" />
+
+    <file src="ffmpeg\Build\Windows10\x64\bin\*.dll"                      target="runtimes\win10-x64\native" />

```

Comment:
I am not planning to create a Nuget stream for FFmpegInterop due to software patent situation. But if you plan to create one, you should include the ARM and ARM64 binaries as well.

Generally, I would rather see nuget stuff in a separate PR. It does not have so much to do with the build script, and I guess it might need more and different testing.

Reply:
I understand there is no intention to publish the NuGet, but this file adds the capacity.
NuGets can be convenient even if they are used in private feeds.

I'll add the ARM* variants when I get to build them successfully (I have some issues with my MINGW installation and haven't succeeded yet).

Reply:
This adds the capacity to do so, even if you don't plan to.
There are many scenarios where NuGets are kept in private feeds, but save hours in build times of dependencies.

I myself would be using this kind of scenario.

It is unrelated to the core of this change, but hope it's not too bad to have here ;)

Reply:
I'll add the ARM* variants in the next push.